liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/UndeployAppMojo.java

```diff
@@ -30,42 +32,37 @@
 /**
  * Undeploy application from liberty server. If no parameters have been defined
  * the mojo will undeploy all applications from the server.
- *
- * @goal undeploy
  */
+@Mojo(name = "undeploy")
+
 public class UndeployAppMojo extends BasicSupport {
     
     /**
      * A file name which points to a specific module's jar | war | ear | eba |
      * zip archive.
-     *
-     * @parameter expression="${appArchive}"
-     * @optional
      */
+    @Parameter(property = "appArchive")
```

Comment:
I don't see the optional "@optional" annotation get converted in Java 5 annotation